File: JAVA/2942. Find Words Containing Character.java
class Solution {
    public List<Integer> findWordsContaining(String[] w, char x) {
        List<Integer> a=new ArrayList<>();
        for(int i=0;i<w.length;i++){
            for(int j=0;j<w[i].length();j++){
                if(w[i].charAt(j)==x){
                    a.add(i);
                    break;
                }
            }
        }
        return a;
    }
}

File: C++/2942. Find Words Containing Character.cpp
class Solution {
public:
    bool is(string s,char x){
        for(char c:s){
            if(c==x)
            return true;
        }
        return false;
    }
    vector<int> findWordsContaining(vector<string>& w, char x) {
        vector<int>a;
        for(int i=0;i<w.size();i++){
            if(is(w[i],x))
            a.push_back(i);
        }
        return a;
    }
};